continuation-local-storage/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for continuation-local-storage v3.2.0
+// Project: https://github.com/othiym23/node-continuation-local-storage
+// Definitions by: Jang-Ho Hwang <https://github.com/rath>, Kei Son <https://github.com/heycalmdown>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare module ContinuationLocalStorage {
+  type Context = any;
+
+  interface Namespace {
+    active(): Context;
+    set<T>(key: string, value: T): T;
+    get(key: string): any;
+    run(callback: any): void;
+    runAndReturn(callback: any): any;
+    bind(callback: any, context?: Context): any;
+    bindEmitter(emitter: NodeJS.EventEmitter): any;
+    createContext(): Context;
+    name: string;
+    context: Context;
+  }
+  function createNamespace(name: string): Namespace;
+  function getNamespace(name: string): Namespace;
+  function destroyNamespace(name: string): void;
+  function reset(): void;
+}
+
+// declare namespace process {
+//   var namespaces: ContinuationLocalStorage.Namespace[];
+// }
```

Comment:
With you solution, I got this
```bash
$ tsc

continuation-local-storage-tests.ts(25,16): error TS2339: Property 'namespaces' does not exist on type 'Process'.
continuation-local-storage-tests.ts(457,33): error TS2339: Property 'namespaces' does not exist on type 'Process'.
index.d.ts(27,16): error TS2671: Cannot augment module 'process' because it resolves to a non-module entity.
index.d.ts(28,26): error TS2503: Cannot find namespace 'ContinuationLocalStorage'.
```

It looks like these resolved issues(https://github.com/Microsoft/TypeScript/issues/7148, https://github.com/Microsoft/TypeScript/issues/7545), but no luck for me with (tsc@2.0.9)

Would I comment the interface out at this time?